css/index.css

```diff
@@ -0,0 +1,548 @@
+    letter-spacing: 0.2px;
+    color: #252B42;
+    margin: 0;
+}
+.teams__title{
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 57px;
+    letter-spacing: 0.2px;
+    color: #252B42;
+    margin: 0;
+}
+.products__desc{
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #737373;
+}
+
+.teams__desc{
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #737373;
+}
+.card{
+    display: grid;
+    grid-template-columns: auto auto auto;
+    margin-top: 50px;
+}
+.card__header{
+    display: flex;
+}
+.card__name{
+    margin-top: auto;
+    margin-bottom: auto;
+}
+.card__desc{
+    text-align: left;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.2px;
+    color: #737373;
+}
+.card__name{
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 24px;
+    letter-spacing: 0.1px;
+    color: #252B42;
+    margin-left: 20px;
+}
+.card__item{
+    border: 1px solid #E8E8E8;
+    padding: 35px 40px;
+    width: fit-content;
+    margin-bottom: 30px;
+}
+.service{
+    text-align: center;
+    margin-top: 100px;
+}
+.questions__desc{
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #737373;
+}
+.service__img{
+    width: 100%;
+    pointer-events: none;
+}
+.service__title{
+    font-size: 30px;
+    font-weight: 700;
+}
+.questions{
+    text-align: center;
+    padding-top: 100px;
+    padding-bottom: 50px;
+    margin-top: 50px;
+    background-color: #F3F3F3;
+}
+.card__item--middle{
+    text-align: left;
+    background-color: white;
+    border-radius: 3px;
+}
+.card__paragraph{
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 24px;
+    letter-spacing: 0.2px;
+    line-height: 3px;
+}
+.card__paragraph--active{
+    color: #FFA62B;
+}
+.card__cover{
+    width: 47px;
+    height: 44px;
+    border: 1px solid #DEDEDE;
+    border-radius: 100%;
+    object-fit: cover;
+}
+.teams{
+    text-align: center;
+    margin-top: 100px;
+}
+.card-teams{
+    display: grid;
+    grid-template-columns: auto auto auto;
+    margin-top: 50px;
+    margin-bottom: 100px;
+}
+.card-teams__item{
+    width: fit-content;
+    border: 2px solid #EEEEEE;
+}
+.cardteams__img{
+    height: 231px;
+    width: 328px;
+    object-fit: cover;
+}
+.teams__name{
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 14px;
+    text-align: center;
+    letter-spacing: 0.1px;
+}
+.teams__pro{
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 14px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #737373;
+}
+.teams__footer{
+    margin-left: -50px;
+}
+.teams__list {
+    display: inline;
+    margin-left: 12px;
+}
+.contact{
+    background-image: url("../img/cover.png");
+    background-size: cover;
```

Comment:
align well your code. you can install an extension called ESLINT in your visual code

Reply:
ESLINT does not work on my vscode but I try to use Prettier.